bluez-generated/README.md

```diff
@@ -3,10 +3,11 @@
 [![crates.io page](https://img.shields.io/crates/v/bluez-generated.svg)](https://crates.io/crates/bluez-generated)
 [![docs.rs page](https://docs.rs/bluez-generated/badge.svg)](https://docs.rs/bluez-generated)
 
-This crate contains async bindings for Bluez.
+This crate contains async D-Bus bindings for BlueZ.
```

Comment:
Maybe the same treatment for the description in Cargo.toml ?

(not for now, but something that I've been pondering):

It's also worth noting that crates.io loads the crate page in two parts, so if you are on a slow connection with no cache, you will notice that it flashes up a page with just the description and then fills in the README.md afterwards. This is a bit janky.

In the back of my mind, I think it might be possible to reduce the jank by repeating the description in README.md **exactly**, but I've not tried it yet. I also don't know how it would interact with badges)

Reply:
Edited to make the first sentence the same as in Cargo.toml.